src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -20,6 +28,16 @@ public class ListCommandTest {
     private Model model;
     private Model expectedModel;
 
+    private ListCommand listCommand;
+    private final Module module = new ModuleBuilder()
+            .withCode("CS2040S").withName("Data Structures and Algorithms")
+            .withTags("Heavy", "Math", "Analysis")
+            .withLectures(TypicalLectures.CS2040S_WEEK_1, TypicalLectures.CS2040S_WEEK_2,
+                    TypicalLectures.CS2040S_WEEK_3, TypicalLectures.CS2040S_WEEK_4,
+                    TypicalLectures.CS2040S_WEEK_5, TypicalLectures.CS2040S_WEEK_6,
+                    TypicalLectures.CS2040S_WEEK_7)
+            .build();
```

Comment:
Instead of duplicating code from TypicalModules.java, could you perhaps change this to `new ModuleBuilder(TypicalModules.CS2040S).build()`?